src/renderer/dx/DxRenderer.cpp

```diff
@@ -1214,6 +1397,51 @@ enum class CursorPaintType
     return S_OK;
 }
 
+// Routine Description:
+// - Paint terminal effects.
+// Arguments:
+// Return Value:
+// - S_OK or relevant DirectX error.
+[[nodiscard]] HRESULT DxEngine::_PaintTerminalEffects()

```

Comment:
This function needs to be marked `noexcept` to get the Static Analysis build to pass